File: SpringBoot/demo/src/main/java/com/example/demo/Repository/MemberRepository.java
package com.example.demo.Repository;

import com.example.demo.Model.Member;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface MemberRepository extends JpaRepository<Member, Integer> {
    Member findMemberById(int id);
}

File: spring-boot-kotlin/converter/src/main/kotlin/com/example/converter/member/repository/MemberRepository.kt
package com.example.converter.member.repository

import com.example.converter.member.entity.Member
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface MemberRepository : JpaRepository<Member, Long>{
    override fun findAll(): MutableList<Member>
}